src/main/java/com/newrelic/agent/instrumentation/verify/AfterEvaluationAction.java

```diff
@@ -98,15 +98,15 @@ public void execute(@NonNull Project project) {
      *
      * Useful in the evaluate phase to add dependencies before the execution phase for only the projects we will actually verify.
      */
-    static boolean projectRequiresVerification(Project project) {
+    public boolean projectRequiresVerification(Project project) {
         return project.getGradle().getStartParameter().getTaskNames().stream()
                 .filter(taskName -> taskName.endsWith(VERIFIER_TASK_NAME))
                 .map(taskName -> getProjectPath(project, taskName))
                 .filter(Objects::nonNull)
                 .anyMatch(projectName -> project.getProjectDir().getPath().startsWith(projectName));
     }
 
-    private static String getProjectPath(Project project, String taskName) {
+    public String getProjectPath(Project project, String taskName) {
```

Comment:
Static is no longer needed

Reply:
Can it still be `private`, though? It doesn't appear to be called directly anywhere but here.

Reply:
good point, fixed!